Base/res/js/Spreadsheet/runtime.js

```diff
@@ -474,6 +474,21 @@ function reflookup(req_lookup_value, lookup_inputs, lookup_outputs, if_missing,
     );
 }
 
+function abs(value) {
+    return Math.abs(value)
+}
```

Comment:
This feels kinda pointless, why yoink this out of the Math namespace?
Generally free functions in here should make sense on a range, not a single value, so I don't really think this function is even needed (as the user can just as easily write `Math.abs`)